volume.go

```diff
@@ -418,3 +416,20 @@ func (v *Volume) RenameVolume(ctx context.Context, newName, volID string) error
 	}
 	return v.client.executeWithRetryAuthenticate(ctx, http.MethodPost, fmt.Sprintf(api.UnityModifyLunURI, volID), lunParams, nil)
 }
+
+// GetMaxVolumeSize - Returns the max size of a volume supported by the array
+func (v *Volume) GetMaxVolumeSize(ctx context.Context, systemLimitID string) (*types.MaxVolumSizeInfo, error) {
+	volumeResp := &types.MaxVolumSizeInfo{}
+	if len(systemLimitID) == 0 {
+		return nil, errors.New("system limit ID shouldn't be empty")
+	}
+	lunURI := fmt.Sprintf(api.UnityAPIGetMaxVolumeSize, systemLimitID, MaximumVolumeSize)
+	err := v.client.executeWithRetryAuthenticate(ctx, http.MethodGet, lunURI, nil, volumeResp)
+	fmt.Println(volumeResp.MaxVolumSizeContent.ID)
+	fmt.Println(volumeResp.MaxVolumSizeContent.Limit)
```

Comment:
@mdutka-dell Do you want to convert these to log statements or remove them if not necessary ?

Reply:
I removed them, thanks